File: 1.Bolum/yerel_bildirim/go/main.go
package main

import "fmt"

func main() {

	var m, n int
	m = 7
	n = ikikat(m)
	fmt.Printf("%d nin iki kati = %d dir", m, n)

}

func ikikat(m int) int {

	var gecici int
	gecici = m * 2
	return gecici

}

File: 1.Bolum/yerel_bildirim/c/main.c
main()
{
    int m, n;
    m = 7;
    n = ikikat(m);
    printf("%d nin iki kati = %d dir", m, n);
}
int ikikat(int k)
{
    int gecici;
    gecici = k * 2;
    return gecici;
}
